silkrpc/commands/eth_api.cpp

```diff
@@ -198,7 +230,7 @@ asio::awaitable<void> EthereumRpcApi::handle_eth_get_block_by_number(const nlohm
 asio::awaitable<void> EthereumRpcApi::handle_eth_get_logs(const nlohmann::json& request, nlohmann::json& reply) {
     auto params = request["params"];
     if (params.size() != 1) {
-        auto error_msg = "invalid eth_getLogs params: " + params.dump();
+        auto error_msg = "invalid handle_eth_get_logs params: " + params.dump();
```

Comment:
`eth_getLogs` was OK here

Reply:
This change seems not homogeneous with the previous ones:
i.e routine handle_eth_get_logs()
the log should contains:
1) handle_eth_get_logs () <-no 
2) eth_get_logs()
3) eth_getLogs()

To have the same rule also in the previous comments

Reply:
Sorry, use `eth_getLogs`

Reply:
ok